scripts/chime8/pipeline/launchers/launch_inference_optuna.sh

```diff
@@ -19,7 +19,7 @@ MSDD_MODEL_PATH=${CHECKPOINTS_DIR}/msdd_v2_PALO_bs6_a003_version6_e53.ckpt
 SCENARIOS="[mixer6]"
 SUBSETS="[dev]"  # can only optimize one subset
 
-SCRIPT_NAME=${NEMO_ROOT}/scripts/chime7/pipeline/run_full_pipeline_optuna.py
+SCRIPT_NAME=${NEMO_ROOT}/scripts/chime7/pipeline/inference_optuna.py
```

Comment:
`inference_optuna.py` file not found? is it `tune_all.py`?

Reply:
I changed it ! Thanks a lot